lblod/frontend-gelinkt-notuleren
# Set-up bestuursperiodes

### Overview
This PR sets-up the `bestuursperiode` ember-data model.
It represents the legislation periode for a specific bestuursorgaan.

Additionally, when creating a new inauguration meeting, 
the correct bestuursorgaan corresponding to the '2024-...' bestuursperiode is now selected.


### Setup
Ensure you are running the latest version of the backend or are proxying to dev.

### How to test/reproduce
- Start the frontend
- Login as any muncipality/OCMW/district that will still exist in the next legislation
- When creating a new IV, ensure the `bestuursorgaan` corresponding to the '2024-...' period is selected.

### Challenges/uncertainties
Should we disable the IV options for?
- Provinces
- Bestuurseenheid that no longer exist.

### Checks PR readiness
